Survey/Survey/Sources/Presentation/Constants/EntryViewType.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  EntryViewType.swift
+//  Survey
+//
+//  Created by Taher on 3/11/22.
+//
+
+import Foundation
+
+enum EntryViewType {
+    case splash
```

Comment:
Same here, missing line break

Reply:
update in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/f10d87bdfe1267e19fe24bb94768bd66860444a7)